internal/output/plain.go

```diff
@@ -22,6 +22,42 @@ import (
 	"github.com/thoas/go-funk"
 )
 
+// PrependedSliceItem wraps an output in a slice and prepends it with a custom string
+type PrependedSliceItem struct {
+	v       interface{}
+	prepend string
+}
+
+// NewPrependedSliceItem wraps an output item in a slice and prepend its output with a custom string instead of a bullet point
+func NewPrependedSliceItem(v interface{}, prepend string) PrependedSliceItem {
+	return PrependedSliceItem{v, prepend}
+}
+
+// PrependString returns the prepended string
+func (po PrependedSliceItem) PrependString() string {
+	return po.prepend
+}
+
+// Value returns the wrapped value
+func (po PrependedSliceItem) Value() interface{} {
+	return po.v
+}
+
+// StackedOutput wraps a slice and ensures that all elements are interpreted individually
+type StackedOutput struct {
+	v []interface{}
+}
+
+// NewStackedOutput wraps a slice and ensures that all elements are interpreted individually but not prepended with bullet points
+func NewStackedOutput(v []interface{}) *StackedOutput {
+	return &StackedOutput{v}
+}
+
+// Stacks returns the wrapped slice
+func (s *StackedOutput) Stacks() []interface{} {
+	return s.v
+}
```

Comment:
I'm confused what this is doing? We specifically did the vertical tables story so that stories like this one could just use that mechanic.

Reply:
As I have an array/slice of different blocks that I have to print.  So, I print the output as a slice, but slices are always interpreted as lists (with bullet points before them).  So, I added this construct that allows us to create such a stacked output.

I had a different idea, that I will create a commit for.  But I think I like this one better...